Modules/Core/Common/test/itkXMLFileOutputWindowTest.cxx

```diff
@@ -20,6 +20,16 @@
 #include "itkXMLFileOutputWindow.h"
 #include "itkTestingMacros.h"
 #include <fstream>
+#include <cstdio>
+#ifdef _WIN32
+#  include <direct.h>
+#  define cwd _getcwd
+#  define OS_SEP "\\"
+#else
+#  include "unistd.h"
+#  define cwd getcwd
+#  define OS_SEP "/"
+#endif
```

Comment:
There may be some more elegant solution to get the working directory and the OS separator.